FirstProject/new-evaluation.html

```diff
@@ -4,65 +4,533 @@
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Tooling <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Browser Engines <select>
+                        <option>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+                </ul>
+            </fieldset>
+        </form>
+
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>NodeJS</legend>
+                <ul style="display: inline-flex; text-align: right">
+                    <li>
+                       Backend frameworks <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Unit Testing <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+
+                    <li>
+                        Templating <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+
+                    <li>
+                        DOM Manipulation <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+                </ul>
+            </fieldset> <br>
+            <input class="submit_button" type="submit" value="SUBMIT" style="float: right; margin-top: 30px; padding: 15px 40px; margin-right: 100px;">
+        </form>
     </div>
-</body>
 
+    <footer class="container-fluid footer_side" style="margin-top: 80px;">
+        <div class="col-md-4">
```

Comment:
These `div` tags are not necessary in this context. You can align the `footer` text using other, simpler methods